trpcultivate_germcollection/src/Plugin/TripalImporter/GermplasmRelationshipImporter.php

```diff
@@ -0,0 +1,1193 @@
+      '#weight' => -99,
+      '#required' => TRUE,
+    ];
+
+    // Options used to search for germplasm/stock names.
+    $options = [
+      'base_table' => 'stock',
+      'column_name' => 'name',
+      'type_column' => 'type_id',
+      'property_table' => 'stock',
+      'type_id' => 0,
+      'match_limit' => 10,
+    ];
+
+    // Get the autocomplete search for the population stock.
+    $form['fieldset_primary_germplasm']['fld_text_primary_germplasm'] = [
+      '#type' => 'textfield',
+      '#autocomplete_route_name' => 'tripal_chado.generic_autocomplete',
+      '#autocomplete_route_parameters' => $options,
+      '#maxlength' => 1000,
+      '#placeholder' => $this->t('Germplasm / Variety / Cultivar'),
+      '#disabled' => FALSE,
+      '#id' => 'population-importer-fld-text-population-entry',
+    ];
+
+    // FIELD: Autocomplete search.
+    // Search germplasm/stock as the primary germplasm.
+    // Relationship Verb.
+    // FIELDSET: Relationship Verb fieldset.
+    $form['fieldset_relationship_type'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Relationship Verb'),
+      '#weight' => -98,
+      '#required' => TRUE,
+    ];
+
+    $term_autocomplete_default = 'Has Member (SIO:000059)';
+
+    $form['fieldset_relationship_type']['fld_select_relationship_verb'] = [
+      '#type' => 'textfield',
+      '#required' => TRUE,
+      '#default_value' => $term_autocomplete_default,
+      '#disabled' => FALSE,
+      '#autocomplete_route_name' => 'tripal.cvterm_autocomplete',
+      '#autocomplete_route_parameters' => ['count' => 10],
+      '#id' => 'population-importer-fld-select-relationship-verb',
+    ];
+
+    // FIELD: Radio buttons.
+    // Set stock position in the relationship.
+    // evi: entry-verb-individual.
+    // ive: individual-verb-entry.
+    $form['fieldset_relationship_type']['fld_radio_stock_position'] = [
+      '#type' => 'radios',
+      '#default_value' => 'evi',
+      '#options' => [
+        'evi' => $this->t('Primary Germplasm as SUBJECT and Related Germplasm as OBJECT of the relationship.'),
+        'ive' => $this->t('Related Germplasm as SUBJECT and Primary Germplasm as OBJECT of the relationship.'),
+      ],
+    ];
+
+    // IMAGE: Population load illustration.
+    $path = base_path() . \Drupal::service('extension.list.module')->getPath('trpcultivate_germcollection');
+    $form['fieldset_relationship_type']['image_illustration'] = [
+      '#markup' => '<div style="margin-top: 20px"><img src="' . $path . '/theme/images/relationship_importer.png" style="max-width: 70%" /></div>',
+    ];
+
+    $form['relationship_toggle'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Related germplasm must already exist'),
+      '#default_value' => 0,
+      '#weight' => -97,
+    ];
```

Comment:
The relationship toggle is currently all by itself where all other form elements are within a fieldset. I would suggest adding it to the file fieldset.

```suggestion
    $form['file']['relationship_toggle'] = [
      '#type' => 'checkbox',
      '#title' => $this->t('Related germplasm must already exist'),
      '#default_value' => 0,
      '#weight' => -97,
    ];
```